frontend/src/components/dashboard/Dashboard.tsx

```diff
@@ -32,7 +23,7 @@ const useStyles = makeStyles((theme) => {
       minWidth: 300,
       width: "100%",
       borderRadius: 5,
-      border: theme.borders.thick,
+      border: `3px solid ${theme.palette.primary.main}`,
```

Comment:
Hmmmmm, did you determine this based on MUI's suggestion?

Reply:
While changing the theme I realized having a custom property "thick"/"thin" in the theme for just 2 files was a bit overkill. All I did here was basically directly putting the css into the "Dashboard" file and removing the generic property from the theme